app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
-
 // Font Colors
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
-
-// Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
```

Comment:
@whmii there isn't always a relationship between the hover button background color and the link hover color and those are the only two use cases I have for that variable.